### ISSUE TITLE
training pkl data

### ISSUE THREAD

**Issue by PROMISE-coder-ai** [SehwanChoi0307/Mask2Map#7]
Hello, would you like to ask whether this work is the same as the training data of mapTRv2? Why do I get an error when I generate pkl file according to the steps? Another problem is that I can also train using the pkl data of MapTRv2, but it does not reach the results given in your paper. I also downloaded the checkpoint file you provided, and the same problem occurred during verification. May I ask what went wrong in this situation.

**Comment by SehwanChoi0307** [comment]
We are using the same pkl as MapTRv2. Additionally, we downloaded a new pkl through the link provided by MapTR and conducted the evaluation using the weight files from GitHub. 
(link file : https://github.com/hustvl/MapTR/releases )
We confirmed the same results as shown in the picture below. If you have downloaded or created a new pkl, you must delete the existing 'nuscenes_map_anns_val.json' file before performing the evaluation. If the order in the val pkl is different, the annotation file must also be changed.

![results](https://github.com/user-attachments/assets/da5e2998-4d39-410e-9768-3a59fd450e4d)
